go-controller/pkg/crd/egressfirewall/v1/register.go

```diff
@@ -0,0 +1,29 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	GroupName          = "k8s.ovn.org"
+	SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1"}
```

Comment:
Redundant question: but are we OK calling the crd version as v1 or should it be v1beta1 or something not quite v1?

Reply:
@abhat probably OK for v1; this is AIUI effectively the same as the SDN bits which have been around forever.

Reply:
ok, awesome. just needed a sanity check there. thanks.